### ISSUE TITLE
Reproduce

### ISSUE THREAD

**Issue by Grace160** [rulixiang/wetr#6]
Hi, thanks for sharing  the very interesting work!
However, when I reproduced, I only achieved the effect of 'miou': 0.6101686126054515  at 20000 iters.
Can you share the code that uses CRF for verification? 
Thanks again and looking forward to your reply!

**Comment by rulixiang** [comment]
Hey, glad you successfully reproduced the code.
For the testing stage, I used multiscale inference (0.5, 0.75, 1, 1.25, 1.5), and the CRF setting is 
```
DenseCRF(
        iter_max=10,  
        pos_xy_std=3,  
        pos_w=3,       
        bi_xy_std=64,  
        bi_rgb_std=5,  
        bi_w=4,  
    )
```
See [Here](https://github.com/rulixiang/vwe/blob/411cb34121b1a641a58171b1bae5f970058d0ae3/v2/crf.py#L105) for the usage of CRF.

Btw, it seems that your result is lower than mine (62.6% mIoU, Tab 2 in paper) . I remember performance at iter 18000 is slightly better.

**Comment by JayeforC** [comment]
> Hey, glad you successfully reproduced the code. For the testing stage, I used multiscale inference (0.5, 0.75, 1, 1.25, 1.5), and the CRF setting is
> 
> ```
> DenseCRF(
>         iter_max=10,  
>         pos_xy_std=3,  
>         pos_w=3,       
>         bi_xy_std=64,  
>         bi_rgb_std=5,  
>         bi_w=4,  
>     )
> ```
> 
> See [Here](https://github.com/rulixiang/vwe/blob/411cb34121b1a641a58171b1bae5f970058d0ae3/v2/crf.py#L105) for the usage of CRF.
> 
> Btw, it seems that your result is lower than mine (62.6% mIoU, Tab 2 in paper) . I remember performance at iter 18000 is slightly better.

Sir, I see the reults from the  ./logs/train_VOC.log are promisingly higher than the result (62.6% mIoU, Table 2 in paper ). Did you use CRF post-processing to generate the ./logs/train_VOC.log?  Look forward to your warmly reply soon.

**Comment by zwyang6** [comment]
> Btw, it seems that your result is lower than mine (62.6% mIoU, Tab 2 in paper) . I remember performance at iter 18000 is slightly better.

Sir, thanks for your interesting and novel work. I tried to reproduce your work with the pretrained model you offered. Here are the most promising results I get:
With Reg loss that works well without bilateralfilte: 
'miou': 0.6003295915381676 for seg score 
'miou': 0.6226548589777818 for aff cam score
'miou': 0.602844093589483  for cam score

which are quite close to the results you showed in the paper.
But when I tried to add the Reg loss with bilateralfilte, the results get worse and are not as positively  good as the results you offered in /afa/logs/voc_train.log.  Coudl you share the very pretrained model that generates /afa/logs/voc_train.log this file ??

**Comment by rulixiang** [comment]
Hey, @JayeforC , you can find the trained models at this [issue](https://github.com/rulixiang/afa/issues/3#issuecomment-1107694274).

**Comment by zwyang6** [comment]
> Hey, @JayeforC , you can find the trained models at this [issue](https://github.com/rulixiang/afa/issues/3#issuecomment-1107694274).

Thank you very much. It works for me. I got the exact  mIou :  0.6603165787468066 as suggested in your paper(Table 2) when i use the wetr_iter_18000.pth you offered. Your efforts are meaningful and inspiring !

**Comment by shulan-s** [comment]
Hey, thanks for your good work. I seem to have encountered some problems. When I copied, I only got "pAcc" effects on the iter 18000: 0.761, "mAcc": 0.852, and "miou": 0.492.
In addition, I want to use remote sensing dataset for training. Can you give the steps of how to build a dataset.
Thanks again and looking forward to your reply!

**Comment by Baconbuilder** [comment]
> > Hey, @JayeforC , you can find the trained models at this [issue](https://github.com/rulixiang/afa/issues/3#issuecomment-1107694274).
> 
> Thank you very much. It works for me. I got the exact mIou : 0.6603165787468066 as suggested in your paper(Table 2) when i use the wetr_iter_18000.pth you offered. Your efforts are meaningful and inspiring !

Hello @JayeforC, could you share how you used the trained model to obtain mIou : 0.6603165787468066 from the paper? I am also not quite sure how to apply CRF for verification. Thank you and I look forward for your reply.
